pkg/utils/environment_variable.go

```diff
@@ -0,0 +1,58 @@
+package utils
```

Comment:
In terms of the naming of this package, it doesn't tell me much. So IMO add a comment at the top of the file what benefit this util gives us or change the name. I vote for comment (:

Reply:
yeah the package name is for all the utils in this project. i will add a comment

Reply:
Thinking about it again, i think that the file `utils/environment_variable.go` is quite self-explanatory. would the comment `This file centralizes functions for extracting values from environment variables` help the reader understand it better? I think it does not.

Reply:
i added comments for each method instead